src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
Maybe it's better to add "opt" to define that it's specific optimal kernel?
resample_opt_onnx_5d.cl?

Reply:
The resample_opt kernel is already saturated. How about moving all 5D related kernels or all linear_onnx kernels, if you separate the files anyway? It is just suggestion.

Reply:
It is a reason why we decided to create a separate kernel. Maybe it makes sense to move all onnx stuff in one file, let's wait till the https://github.com/openvinotoolkit/openvino/pull/12977 is merged and see how all of it be together..

Reply:
> It is a reason why we decided to create a separate kernel. Maybe it makes sense to move all onnx stuff in one file, let's wait till the #12977 is merged and see how all of it be together.

https://github.com/openvinotoolkit/openvino/pull/12977 has been merged. How are you going to do this PR?

Reply:
onnx will be moved in a separate kernel. we begin working on that.